vertical-pod-autoscaler/updater/priority/update_priority_calculator.go

```diff
@@ -132,6 +166,11 @@ func (list byPriority) Swap(i, j int) {
 	list[i], list[j] = list[j], list[i]
 }
 func (list byPriority) Less(i, j int) bool {
-	// reverse ordering, highest priority first
-	return list[i].priority > list[j].priority
+	// Reverse ordering, highest priority first.
+	// 1. If any container wants to grow, the pod takes precedence.
```

Comment:
I find this disturbing - why those with shrinking resources shouldn't be first (e.g. to free resources for growing ones to use)?